frappe/app.py

```diff
@@ -218,14 +218,13 @@ def make_form_dict(request):
 		args.update(request.args or {})
 		args.update(request.form or {})
 
-	if not isinstance(args, dict):
-		frappe.throw(_("Invalid request arguments"))
+	if isinstance(args, dict):
```

Comment:
are you sure this is what you want? 

This is just silently ignoring and throwing away request body if it's non-dict?

Reply:
Also Frappe API calls map to a function. Array of input to a function would be what exactly? Something like this?


```
def handle(**input):
    pass
```

Reply:
Yes, this is throwing away request body if it is non-dict.

I have created a whitelist function in which i want array in request, but it throwing an error,

Request
![image](https://user-images.githubusercontent.com/109956567/209153466-0575c7d0-981d-4f51-95a5-9d51d9891d13.png)

Response
![image](https://user-images.githubusercontent.com/109956567/209152684-0d403001-7bb5-49f0-b45c-603b2c27ff65.png)

What would you suggest if i want array in api request? @ankush

Reply:
I don't get the point of accepting a request and throwing away the input dict. Care to elaborate further as to why you need this?